USERS.md

```diff
@@ -537,6 +537,11 @@ Users (Alphabetically)
       U: CNI, Overlay networking
       Q: @Madhu CS
 
+    * N: Santa Claus & the Elves
+      D: All our infrastructure to process children's letters and whishes, toy making and delivery, distributed over multiple clusters around the world, is now powered by Cilium.
```

Comment:
```suggestion
      D: All our infrastructure to process children's letters and wishes, toy making, and delivery, distributed over multiple clusters around the world, is now powered by Cilium.
```

Reply:
Thanks! Santa says you'll get an additional present next year.